provider/datagen/src/databake.rs

```diff
@@ -262,37 +259,143 @@ impl DataExporter for BakedDataExporter {
             log::info!("{}", crate_name);
         }
 
-        move_out!(self.mod_files)
+        let mut mod_files = move_out!(self.mod_files).into_inner().expect("poison");
+        for (_, mods) in mod_files.iter_mut() {
+            mods.sort();
+        }
+
+        let mods = mod_files
+            .remove(&PathBuf::new())
+            .expect("root exists")
+            .into_iter()
+            .dedup()
+            .map(|p| p.parse::<TokenStream>().unwrap());
+
+        let mut marker_data_feature = move_out!(self.marker_data_feature)
             .into_inner()
-            .expect("poison")
-            .into_par_iter()
-            .try_for_each(|(path, mut mods)| {
-                mods.sort();
-                let mods = mods
-                    .into_iter()
-                    .dedup()
-                    .map(|p| p.parse::<TokenStream>().unwrap());
-                let maybe_decl = if path.as_os_str().is_empty() {
-                    quote! {
+            .expect("poison");
+        marker_data_feature.sort();
+        let marker_data_feature = marker_data_feature
+            .into_iter()
+            .map(|(marker_str, data_str, feature_str)| {
+                (
+                    marker_str.parse::<TokenStream>().unwrap(),
+                    data_str.parse::<TokenStream>().unwrap(),
+                    feature_str.parse::<TokenStream>().unwrap(),
+                )
+            })
+            .collect::<Vec<_>>();
+
+        let any_cases = marker_data_feature.iter().map(|(marker, data, feature)| {
+            if marker.to_string() == ":: icu_datetime :: provider :: calendar :: DateSkeletonPatternsV1Marker" {
```

Comment:
Praise: Glad that you got this part to work.

Nit: Add a TODO pointing to #1678